eng/xplat-job.yml

```diff
@@ -35,21 +35,36 @@ jobs:
     enableMicrobuild: ${{ parameters.enableMicrobuild }}
 
     pool:
+
+      # Public Linux Build Pool
       ${{ if and(eq(parameters.osGroup, 'Linux'), eq(variables['System.TeamProject'], 'public')) }}:
-        name: dnceng-linux-external-temp
+        name:  NetCorePublic-Int-Pool
+        queue: BuildPool.Ubuntu.1604.Amd64.Open
+
+      # Official Build Linux Pool
       ${{ if and(eq(parameters.osGroup, 'Linux'), ne(variables['System.TeamProject'], 'public')) }}:
         name: dnceng-linux-internal-temp
+
       # FreeBSD builds only in the internal project
       ${{ if and(eq(parameters.osGroup, 'FreeBSD'), ne(variables['System.TeamProject'], 'public')) }}:
         name: dnceng-freebsd-internal
+
+      # Public OSX Build Pool
       ${{ if and(eq(parameters.osGroup, 'OSX'), ne(variables['System.TeamProject'], 'public')) }}:
```

Comment:
The OSX and Windows comments need to be swapped across public/internal. Could you re-order these to make the public/internal order consistent while you're at it? Thanks!

Reply:
Will create another pr with these changes.